terraform/monitoring/dashboards/frontend_dashboard.json

```diff
@@ -0,0 +1,153 @@
+{
+  "displayName": "Frontend Service Dashboard",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "CPU limit utilization for stackdriver-sandbox, frontend",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"kubernetes.io/container/cpu/limit_utilization\" resource.type=\"k8s_container\" resource.label.\"cluster_name\"=\"stackdriver-sandbox\" metadata.user_labels.\"app\"=\"frontend\"",
+                  "aggregation": {
+                    "perSeriesAligner": "ALIGN_MEAN"
+                  },
+                  "secondaryAggregation": {}
+                },
+                "unitOverride": "1"
+              },
+              "plotType": "LINE",
+              "minAlignmentPeriod": "60s"
+            }
+          ],
+          "timeshiftDuration": "0s",
```

Comment:
See previous comment about removing all default ("0s") instances of this.

Reply:
Changed :)